config/blobs.yml

```diff
@@ -2,7 +2,6 @@ haproxy/haproxy-2.7.8.tar.gz:
   size: 4176647
   object_id: 2c695f4b-8032-463c-569c-a2a44c62e152
   sha: sha256:15f2276971bbba8c47d86cc82ebfc6ec33e3aef2e4565058b2e4950c07b8e75c
-jq/jq-1.6-linux64.tgz:
-  size: 1708805
-  object_id: cb77c7f7-d8d5-45d5-6117-f49a92bafb1f
-  sha: sha256:2074c8209a0eebe4b9c02be3c405614b475b1bd84fbbb1ca76135f7d0cfcc009
+jq/jq-1.7-linux-amd64.tgz:
+  size: 992924
+  sha: sha256:6c640db360b3a989a4a5f49dfb38128b9f11726da2f8295455e67917e1aa2e32
```

Comment:
Can you `bosh upload-blobs` for this as well?

Reply:
I totally ran that but forgot to check in the changes that it makes to `config/blobs.yml`. 😅 

I uploaded the blob in: https://github.com/cloudfoundry/routing-release/pull/358/commits/f9b84a011dbf77ed9ef62d3e31f3e538f7f4f895

Reply:
Should be good now. I checked this out on a different machine:

```
~/workspace/routing-release |bump-jq-to-1.7 ✔|
$ gup
remote: Enumerating objects: 4, done.
remote: Counting objects: 100% (4/4), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 4 (delta 2), reused 4 (delta 2), pack-reused 0
Unpacking objects: 100% (4/4), 531 bytes | 265.00 KiB/s, done.
From github.com:cloudfoundry/routing-release
   cc33709d..f9b84a01  bump-jq-to-1.7 -> origin/bump-jq-to-1.7
Updating cc33709d..f9b84a01
Fast-forward
 config/blobs.yml | 1 +
 1 file changed, 1 insertion(+)

bosh: null | cf:  (xibalba)
~/workspace/routing-release |bump-jq-to-1.7 ✔|
$ bosh sync-blobs
Blob download 'jq/jq-1.7-linux-amd64.tgz' (993 kB) (id: 03fca711-7950-4545-46ac-cf0c82c24d2c sha1: sha256:6c640db360b3a989a4a5f49dfb38128b9f11726da2f8295455e67917e1aa2e32) started
Blob download 'jq/jq-1.7-linux-amd64.tgz' (id: 03fca711-7950-4545-46ac-cf0c82c24d2c) finished

Succeeded

bosh: null | cf:  (xibalba)
~/workspace/routing-release |bump-jq-to-1.7 ✔|
$ ll blobs
total 16
drwxrwxr-x  4 pivotal pivotal 4096 Oct 12 19:06 ./
drwxrwxr-x 16 pivotal pivotal 4096 Oct 12 19:01 ../
drwxrwxr-x  2 pivotal pivotal 4096 Oct 12 19:01 haproxy/
drwxrwxr-x  2 pivotal pivotal 4096 Oct 12 19:06 jq/

bosh: null | cf:  (xibalba)
~/workspace/routing-release |bump-jq-to-1.7 ✔|
$ tree blobs
blobs
├── haproxy
│   └── haproxy-2.7.8.tar.gz
└── jq
    └── jq-1.7-linux-amd64.tgz

2 directories, 2 files
```